.github/workflows/go.yml

```
@@ -300,18 +300,22 @@ jobs:
         with:
           go-version: ${{ matrix.go }}
           cache: true
-          cache-dependency-path: go/go.sum
+          cache-dependency-path: go/go.sum      
       - name: Brew Install Arrow and pkg-config
         shell: bash
         run: brew install apache-arrow pkg-config
       - name: Install staticcheck
         run: go install honnef.co/go/tools/cmd/staticcheck@${{ matrix.staticcheck }}
       - name: Build
         shell: bash
-        run: ci/scripts/go_build.sh $(pwd)
+        run: |
+          export PKG_CONFIG_PATH=/usr/local/Cellar/openssl@3/3.0.8/lib/pkgconfig:$PKG_CONFIG_PATH
```

This should update it for all following steps by setting it once :)
```suggestion
           echo "PKG_CONFIG_PATH=/usr/local/Cellar/openssl@3/3.0.8/lib/pkgconfig:$PKG_CONFIG_PATH" >> $GITHUB_ENV
```